### DISCUSSION TITLE
CDN worker name incorrectly generated

### DISCUSSION THREAD

**Discussion by oli414** [mozilla/hubs#3732] [Hubs Cloud Questions] [answered]
I disabled the CDN through Cloudflare last week and wanted to reenable it again today, but the admin page seems to incorrectly generate the worker name.

The workers names are currently `hubs-example-com-proxy` and `hubs-example-com-cors-proxy` but the admin page now suggests that they should be `-proxy` and `-cors-proxy`. In Cloudflare I'm not allowed to start the worker name with a dash.

The incorrect names are also used for the test links, and the generated code, I'm not allowed to save the changes as it can't find the workers by those names e.g. `https://-cors-proxy.example.workers.dev/https://www.mozilla.org` and `https://-proxy.example.workers.dev/hubs/pages/latest/whats-new.html`

![image](https://user-images.githubusercontent.com/2348094/105160637-c7863d80-5b10-11eb-8313-9d784c979d46.png)

Something is wrong, how can I make sure that it works again?

**Comment by robinkwilson** [comment, marked as answer]
Are you using a custom client? This looks like an issue we saw many months back. This happens when your account is not an admin (you can check if you see any of the avatar/scenes/account data populating in the other menus). 

If you do have a custom client, I'd merge the latest hubs-cloud branch in. 

Alternatively, is this on DigitalOcean?

If you are planning on having any videos streaming through your instance, please do NOT setup these Cloudflare workers, they can block you unexpectedly.

**Reply by oli414** [reply]
Ah yes, this is a custom client (I am the admin of the instance), I figured I wouldn't need to do that as it did work for me earlier, but that's a good idea.

> If you are planning on having any videos streaming through your instance, please do NOT setup these Cloudflare workers, they can block you unexpectedly.

Oh good call in that case I'll leave it off for now as we do need that. Thanks a lot!
